docs/multicluster/multi-cluster.md

```diff
@@ -317,34 +317,14 @@ The following is the GTP CRD definition.
 
 ```yml
```

Comment:
This YAML should not be part of the documentation, it needs to be link to a file